client/src/main/java/io/obswebsocket/community/client/message/request/config/CreateProfileRequest.java

```diff
@@ -0,0 +1,30 @@
+package io.obswebsocket.community.client.message.request.config;
+
+import io.obswebsocket.community.client.message.request.Request;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.ToString;
+
+@Getter
+@ToString(callSuper = true)
+public class CreateProfileRequest extends Request {
```

Comment:
This one and `RemoveProfile` could extend an abstract `Profile` request with the common prop